torch_em/data/sampler.py

```diff
@@ -85,13 +85,18 @@ class MinInstanceSampler:
     def __init__(
         self,
         min_num_instances: int = 2,
-        p_reject: float = 1.0
+        p_reject: float = 1.0,
+        min_size: int = None
```

Comment:
This should be `min_size: Optional[int] = None`. And then you need to include the `Optional` import on top.